githubcontent/bolts.py

```diff
@@ -0,0 +1,54 @@
+from bs4 import BeautifulSoup
+import requests
+import uuid
+import json
+
+from utils import GithubScraper
+scraper = GithubScraper()
+
+def get_bolts():
+    documents = []
+    bolts = ['00-introduction','01-messaging','02-peer-protocol','03-transactions','04-onion-routing',
+                '05-onchain','07-routing-gossip','08-transport',
+                '09-features','10-dns-bootstrap','11-payment-encoding'
+                ]
+
+    base_url = 'https://github.com/lightning/bolts/blob/master/{}.md'
+    print("Getting links for lightning bolts")
+    bolt_links = scraper.get_github_urls(base_url,bolts)
+    for link in bolt_links:
+        data = requests.get(link).text
+        soup = BeautifulSoup(data,'html.parser')
+        title = soup.find('h1',dir='auto').get_text()
+        body = soup.find('article').get_text()
+        body_type = "md"
+        authors = []
```

Comment:
I don't like leaving this empty. Maybe something like this?
```suggestion
        authors = ["Spec"]
```